invenio/modules/documents/config.py

```diff
@@ -36,5 +34,5 @@
     'model': 'Document',
     'host': "localhost",
     'port': 27017,
-    'database': config.CFG_DATABASE_NAME,
+    'database': LocalProxy(lambda: cfg['CFG_DATABASE_NAME']),
```

Comment:
I would do the encoding here instead https://github.com/CERNDocumentServer/cds-ops/pull/99/files#diff-5d922fafdb7869ccd68b48a7dce881e3R37.